pandas/core/dtypes/common.py

```diff
@@ -832,6 +823,10 @@ def is_any_int_dtype(arr_or_dtype):
 
     This function is internal and should not be exposed in the public API.
 
+    null-able Integer support
```

Comment:
null-able -> Nullable.

Also, I think that should go inside the `versionchanged` directive, not above it.